src/main/flight/mixer_tricopter.c

```diff
@@ -571,7 +571,7 @@ static void tailTuneModeThrustTorque(struct thrustTorque_t *pTT, const bool isTh
     case TT_WAIT_FOR_DISARM:
         if (!ARMING_FLAG(ARMED))
         {
-            float averageServoAngle = pTT->servoAvgAngle.sum / 10.0f / pTT->servoAvgAngle.numOf;
+            float averageServoAngle = pTT->servoAvgAngle.sum / 10.0f / pTT->servoAvgAngle.numOf - 90.0f;
             if (averageServoAngle > 0.5f && averageServoAngle < 40.f) { // TODO: Too wide? or totally unnecessary?
```

Comment:
The best limits should come from the reasonable limits for the factor.Do you have any intuitive idea about that?To get a higher factor, the prop drag must be reduced, and vice verse. There must be some limits to how good/bad a prop can be. Plus a small margin for bad bench tuning. 






On Tue, May 3, 2016 at 10:34 PM -0700, "lkaino" <notifications@github.com> wrote:












In src/main/flight/mixer_tricopter.c:
> @@ -571,7 +571,7 @@ static void tailTuneModeThrustTorque(struct thrustTorque_t *pTT, const bool isTh
>      case TT_WAIT_FOR_DISARM:
>          if (!ARMING_FLAG(ARMED))
>          {
> -            float averageServoAngle = pTT->servoAvgAngle.sum / 10.0f / pTT->servoAvgAngle.numOf;
> +            float averageServoAngle = pTT->servoAvgAngle.sum / 10.0f / pTT->servoAvgAngle.numOf - 90.0f;
>              if (averageServoAngle > 0.5f && averageServoAngle < 40.f) { // TODO: Too wide? or totally unnecessary?



Could you just for clarity use > 90.5f here and remove the - 90.0f on the line above. What happens with values [90, 90.5]? The max limit can be increased to 90+50 (TRI_TAIL_SERVO_MAX_ANGLE).



—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub